src/NServiceBus.Persistence.ServiceFabric/SagaPersister/SagaPersister.cs

```diff
@@ -28,13 +24,6 @@ public Task Complete(IContainSagaData sagaData, SynchronizedStorageSession sessi
                 {
```

Comment:
Pretty sure we do, if the entry was modified from the point we last saw it we need to retry